homeassistant/components/awattar/const.py

```diff
@@ -0,0 +1,17 @@
+"""Constants for the aWATTar integration."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+from typing import Final
+
+DOMAIN: Final = "awattar"
+CONF_COUNTRY: Final = "awattar_country"
```

Comment:
No need to create a new const, we have `homeassistant.const.CONF_COUNTRY`

Reply:
done.